Justfile

```diff
@@ -1,29 +1,44 @@
+set dotenv-load
+
+arch := `uname -m | sed 's/amd64/x86_64/' | sed 's/arm64/aarch64/'`
+
 build:
 	yarn --cwd ./frontend/website build
 	cargo build --release
 
+build-container: env-backup
+	docker run --rm -v $(pwd):/pwd -w /pwd ghcr.io/scuffletv/base-build:1.66.1 just build
+
+env-backup:
+	test -f .env && (\
+		mv .env .env.bak \
+	) || true
+
 format:
 	yarn --cwd ./frontend/website format
 	yarn format
-	cargo fmt
+	cargo fmt --all
 	cargo clippy --fix --allow-dirty --allow-staged
+	cargo clippy --fix --allow-dirty --allow-staged --package player --target wasm32-unknown-unknown
```

Comment:
Does this not double-run for `player`? First from the first 
clippy command and then from the second.

Reply:
Nope, since we have default members which excludes player